src/applications/survivors-benefits/config/chapters/02-claimant-information/claimantInformation.js

```diff
@@ -0,0 +1,78 @@
+import {
+  fullNameSchema,
+  titleUI,
+  ssnUI,
+  ssnSchema,
+  dateOfBirthSchema,
+  dateOfBirthUI,
+  yesNoSchema,
+  yesNoUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import { seriouslyDisabledDescription } from '../../../components/formDescriptons/index';
+import { benefitsIntakeFullNameUI } from './helpers';
+
+/** @type {PageSchema} */
+export default {
+  title: 'Claimant information',
+  path: 'claimant/information',
+  uiSchema: {
+    ...titleUI("Claimant's relationship to the Veteran"),
+    claimantRelationship: {
+      'ui:title': "What is the claimant's relationship to the Veteran?",
+      'ui:widget': 'radio',
+      'ui:options': {
+        labels: {
```

Comment:
I've added a new utils/labels file, can you move these there? It will help when writing tests to use the same object.